pkg/controller/master-controller-manager/rbac/sync_resource.go

```diff
@@ -70,6 +74,9 @@ func (c *resourcesController) syncProjectResource(item *resourceToProcess) error
 			if err := c.ensureRBACRoleBindingForClusterAddons(projectName, item.metaObject, item.clusterProvider); err != nil {
 				return fmt.Errorf("failed to sync RBAC RoleBinding for %s resource for %s cluster provider in namespace %s, due to = %v", item.gvr.String(), item.clusterProvider.providerName, item.metaObject.GetNamespace(), err)
```

Comment:
The `, due to = ` can be replaced with the more common `: `, so that error messages read "failed: <reason>".